apps/nextjs/.env.example

```diff
@@ -1,6 +1,10 @@
 GOOGLE_ID=<copy gogole client id>
 GOOGLE_SECRET=<copy google secret>
-AZURE_AD_B2C_TENANT_NAME=<copy the B2C tenant name here from Step 1>
-AZURE_AD_B2C_CLIENT_ID=<copy Application (client) ID here from Step 2>
-AZURE_AD_B2C_CLIENT_SECRET=<copy generated secret value here from Step 2>
-NEXT_PUBLIC_EXTENSION_ID=<copy from chrome://extensions when inspecting your ext>
\ No newline at end of file
+AZURE_AD_CLIENT_ID=<copy Application (client) ID here>
+AZURE_AD_CLIENT_SECRET=<copy generated client secret value here>
+AZURE_AD_TENANT_ID=<copy the tenant id here>
+NEXT_PUBLIC_EXTENSION_ID=<copy from chrome://extensions when inspecting your ext>
+PORT=3000
+HOST=localhost
```

Comment:
@juliusmarminge I added PORT & HOST to the example .env

Reply:
I'm pretty sure the PORT is polyfilled by Next Dev (so it's always correct)

Reply:
I was wrong on this one